pycondor/job.py

```diff
@@ -439,7 +440,10 @@ def submit_job(self, submit_options=None):
         if submit_options is not None:
             command += ' {}'.format(submit_options)
         command += ' {}'.format(self.submit_file)
-        proc = subprocess.Popen([command], stdout=subprocess.PIPE, shell=True)
+        proc = subprocess.Popen(shlex.split(command),
+                                stdout=subprocess.PIPE,
+                                stderr=subprocess.PIPE,
+                                shell=True)
```

Comment:
Thanks for the update Karl! If it's not too much trouble, could you please remove the `shell=True` lines? It's causing some issues with the submit scripts on mac / linux. After that this PR should be good to merge!

Reply:
No problem, done.